CHANGELOG.md

```diff
@@ -25,6 +25,18 @@ All notable changes to the Wazuh app project will be documented in this file.
 ### Removed
 
 - Removed the setting home:useNewHomePage from the advanced settings because the views are not finished. [#282](https://github.com/wazuh/wazuh-dashboard/pull/282)
+- Reduced the size of the loading logo [#373](https://github.com/wazuh/wazuh-dashboard/pull/373)
+- Changed the order of the "Dashboard management" and "Indexer management" sections [#372](https://github.com/wazuh/wazuh-dashboard/pull/372)
+
+### Removed
```

Comment:
Please check the consistency of the changelog. It seems the Removed section is duplicated and it has an additional entry.

Here's the 4.10.0 version changelog:
https://github.com/wazuh/wazuh-dashboard/blob/4.10.0/CHANGELOG.md

Reply:
Solved in https://github.com/wazuh/wazuh-dashboard/pull/447/commits/179429f12f84afc5c87e37263c06e1408cd03368